website/content/partials/pki-double-migration-bug.mdx

```diff
@@ -0,0 +1,22 @@
+### PKI Storage Migration Revives Deleted Issuers
+
+In Vault 1.11, a new storage layout was created to support multiple issuer
+within a single mount, creating storage version 1. In Vault 1.11.6, 1.12.2,
+and 1.13.0, a bug was fixed wherein (due to incorrect ordering of writes),
+the CA chain could be incorrect, due to writes failing when too many mounts
+attempted migration at the same time; this introduced storage version 2.
+
+#### Impacted Versions
```

Comment:
```suggestion
```

I would remove the extra heading. It doesn't add clarity and tbh, the text that follows seems more like a continuation of the issue description than a discussion of version-specific impact.

Reply:
@schavis All of these known issues have `impacted version` headings, so I think we should keep it for consistency as I think users skim it?

Reply:
```suggestion
#### Affected versions
```